src/libs/shim-interface/src/lib.rs

```diff
@@ -49,14 +52,18 @@ fn get_uds_with_sid(short_id: &str, path: &str) -> Result<String> {
             "sandbox with the provided prefix {short_id:?} is not found"
         )),
         1 => {
+            // FIXME: jailed case seems to fail
             // One element and only one exists.
-            Ok(format!(
-                "unix://{}",
-                kata_run_path
+            let p = match jailed_path {
```

Comment:
that's true -- we thought it would be better if all container-related files are inside the jail, but, indeed, we can keep the monitor shim at the generic (non-jailed) dir.